src/itemAction/index.unit.tsx

```diff
@@ -77,5 +77,17 @@ describe('ItemAction', () => {
       expect(wrapper.find(Loader).exists()).toBe(true)
       expect(wrapper.find(Loader).prop('done')).toBe(true)
     })
+
+    it('Should render an highlighted ItemAction by default', () => {
```

Comment:
nit: for consistency, use lowercase everywhere "should" (or uppercase below)

Reply:
Done